src/includes/set-environment/unity.mdx

```diff
@@ -0,0 +1,6 @@
+```csharp
+using Sentry;
+
+// Add this to the SDK initialization callback
+options.Environment = "production";
```

Comment:
I have mixed feelings about duplicating every single code snippet to avoid having F# in the selector.

At this point we might as well delete the F# examples.

Ideally tooling (the docs) would allow us to exclude a snippet lang from the code snippet file instead of us working around this. @PeloWriter thoughts?